### PR TITLE
feat: added publish profile confirmation popup

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

**Review by hansac91 — APPROVED** [review]

File: src/components/TogglePublishPopup/TogglePublishPopup.module.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  background: var(--dark-03, #161a1f);
+  z-index: 998;
+}
+
+.container {
+  position: fixed;
+  width: 30%;
+  max-height: 100%;
+  overflow-y: auto;
+  border-radius: 10px;
```
Comment (mmcode9407) [inline src/components/TogglePublishPopup/TogglePublishPopup.module.scss:17]: you use `border-radius` second time below

File: src/components/TogglePublishPopup/TogglePublishPopup.module.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  background: var(--dark-02, #0f1217);
+  box-shadow: 0px 0px 30px 0px rgba(125, 84, 241, 0.15);
+  z-index: 999;
+  top: 45%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border-radius: 5px;
```
Comment (mmcode9407) [inline src/components/TogglePublishPopup/TogglePublishPopup.module.scss:24]: here

**Review by mmcode9407 — CHANGES_REQUESTED** [review]
image
The popup is a bit transparent but I don't see the css property responsible for it. Is this problem only with me?

**Review by mmcode9407 — APPROVED** [review]
Nice! Just delete unused imports and props in `ProfileTopBar` before you merge :)

**Review by Sumick — APPROVED** [review]

**Pull request merged**